### ISSUE TITLE
gometalinter deactivated, cannot locate ''

### ISSUE THREAD

**Issue by salarali** [alecthomas/SublimeLinter-contrib-gometalinter#17]
Since upgrading to SublimeLinter to 4.0, gometalinter doesnt work.

All I get is this warning:
`SublimeLinter: SublimeLinter.sublime-package:1063: WARNING: gometalinter deactivated, cannot locate ''
`

SublimeLinter Version: 4.0.6
GoMetaLinter Version: 4.0.4

My config:

```
// SublimeLinter Settings - User
{
	"debug": true,
	"linters": {
		"gometalinter": {
		    "@disable": false,
		    "args": [
		        "--disable-all",
		        "--enable=vet",
		        "--enable=vetshadow",
		        "--enable=golint",
		        "--enable=ineffassign",
		        "--enable=goconst",
		        "--enable=deadcode",
		        "--enable=megacheck"
		    ],
		    "excludes": []
		},
	}
}
```

**Comment by alecthomas** [comment]
I think this is an issue with SublimeLinter not this plugin.

**Comment by salarali** [comment]
You were right. Apparently, SublimeLinter did not have the path for gometalinter. Also this error changed to `WARNING: gometalinter deactivated, cannot locate 'gometalinter'` in the SublimeLinter 4.1 which helped me narrow down the problem.
